tests/chainer_tests/links_tests/activation_tests/test_simplified_dropconnect.py

```diff
@@ -181,6 +183,42 @@ def test_serialization(self):
         self.assertEqual((w1 == w2).all(), True)
 
 
+class TestSimplifiedDropconnectNotBatchwiseMask(unittest.TestCase):
+
+    in_shape = (3,)
+    out_size = 2
+    ratio = 0.5
+
+    def setUp(self):
+        in_size = numpy.prod(self.in_shape)
+
+        self.link = links.SimplifiedDropconnect(
+            in_size, self.out_size,
+            initialW=chainer.initializers.Normal(1, numpy.float32),
+            initial_bias=chainer.initializers.Normal(1, numpy.float32))
+        self.link.cleargrads()
+
+        x_shape = (4,) + self.in_shape
+        self.x = numpy.ones(x_shape).astype(numpy.float32)
+
+    def check_forward(self, x_data):
+        x = chainer.Variable(x_data)
+        y = self.link(x, False, True)
+
+        # check mask equality here.
+        testing.assert_allclose(y.data[0], y.data[1])
+        testing.assert_allclose(y.data[0], y.data[2])
+        testing.assert_allclose(y.data[0], y.data[3])
```

Comment:
If possible, I would like you to check that the values satisfies not only the constraint that they are same but also their values are expected ones.